auth/Cargo.toml

```diff
@@ -9,28 +9,30 @@ anyhow = { workspace = true }
 async-trait = { workspace = true }
```

Comment:
I might be wrong, but we should be able to undo all of the changes in this file right? Since the layer moved to gateway.

Reply:
Indeed! The remaining ones are from cargo-sort.